WooCommerce/Classes/POS/Presentation/PointOfSaleEntryPointView.swift

```diff
@@ -18,11 +18,13 @@ struct PointOfSaleEntryPointView: View {
          currencyFormatter: CurrencyFormatter) {
         self.hideAppTabBar = hideAppTabBar
 
+        let analytics = PointOfSaleAnalytics(analyticsProvider: PointOfSaleTracksProvider())
```

Comment:
Does it make sense to declare these here? Or should we go one level up and do so in the `HubMenuViewModel`, since acts as a composition root for the rest of dependencies?